docs/walkthrough.md

```diff
@@ -89,4 +89,38 @@ kubectl port-forward svc/keda-http-add-on-interceptor-proxy -n ${NAMESPACE} 8080
 curl -H "Host: myhost.com" localhost:8080/path1
 ```
 
+### Integrating HTTP Add-On Scaler with other KEDA scalers
+
+For scenerios where you want to integrate HTTP Add-On scaler with other keda scalers, you can set the SkipScaledObjectCreation annotation to true on your HTTPScaledObject.  The reconciler will then skip the KEDA core ScaledObject creation which will allow you to create your own ScaledObject and add http scaler as one of your triggers.  Please note that you should ensure the ScaledObject is created with a different name to the HTTPScaledObject to ensure your ScaledObject is not removed by the reconciler.
+
+It is reccomended that you first deploy your HTTPScaledObject with no annotation set in order to obtain the latest trigger spec to use on your own managed ScaledObject.
+
+Step 1, first deploy your HTTPSCaledObject with annotation set to false
+
+```console
+annotations:
+  skipScaledObjectCreation: false
+```
+
+Step 2, take a copy of the current generated external-push trigger spec on the generated ScaledObject.  Please find example below, however this spec is likely to change on future releases.
+
+```console
+  triggers:
+  - type: external-push
+    metadata:
+      hosts: example-service
+      pathPrefixes: ""
+      scalerAddress: keda-http-add-on-external-scaler.keda:9090
+```
+
+Step 3, update the skipScaledObjectCreation annotation to true and re-deploy.   This will remove the ScaledObject and allow you to then create your own.
```

Comment:
```suggestion
3. Apply the `skipScaledObjectCreation` annotation with `true` and apply the change. This will remove the originally created `ScaledObject` allowing you to create your own.
```